study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilderCli.java

```diff
@@ -73,6 +76,7 @@ private void run(String[] args) throws Exception {
         options.addOption(null, OPT_EMAIL_KEYS, true, "comma-separated email keys, only create/update emails with these keys");
         options.addOption(null, OPT_RUN_TASK, true, "run a custom task");
         options.addOption(null, OPT_PATCH, false, "run patches for a study");
+        options.addOption(null, OPT_PROCESS_TRANSLATIONS, false, "run the process of translations assignment for all study languages");
```

Comment:
When this option is specified the translations references will be resolved automatically:
`StudyBuilder pancan --vars ../../configs/vars_pancan.conf -substitutions studies/pancan/subs.conf -process-translations`